### ISSUE TITLE
Will xtb4stda still support new elements?

### ISSUE THREAD

**Issue by ironlossli** [grimme-lab/xtb4stda#15]
**Is your feature request related to a problem? Please describe.**
Is this project still being updated?

**Describe the solution you'd like**
 I wish xtb4stda could include lanthanides.

**Comment by mdewergi** [comment]
Yes, the project is still up but they are no plan to include lanthanides any time soon...

**Comment by ironlossli** [comment]
Exciting news, looking forward to your further work.
